packages/clay-css/src/scss/variables/_panels.scss

```diff
@@ -145,17 +172,30 @@ $panel-unstyled-header-link: () !default;
 $panel-unstyled: () !default;
 $panel-unstyled: map-deep-merge(
 	(
-		bg: transparent,
+		background-color: transparent,
 		border-radius: 0,
 		border-width: 0,
-		header-border-color: $gray-500,
-		header-border-style: solid,
-		header-border-width: 0 0 1px 0,
-		header-margin-bottom: 1rem,
-		header-collapsed-border-color: $gray-500,
+		margin-bottom: 1.5rem,
+		header: (
+			border-color: $gray-500,
+			border-radius: 0,
+			border-style: solid,
+			border-width: 0 0 1px 0,
+			padding-left: 0,
```

Comment:
@pat270 in theory who should remove the padding here shouldn't be `flush` with the panel group? could adding this as a default break any other behavior that spacing is expected?

Reply:
Originally I was expecting `panel-group-flush` to remove the padding, but Lexicon always has the text aligned with the header border. If we want it to adhere to Lexicon spec without `panel-group-flush` we should remove it here.

Reply:
Makes sense, so we can treat this as a bug.